platforms/android/xwalk.gradle

```diff
@@ -51,10 +51,10 @@ def getConfigPreference(name, defaultValue) {
 
 // Set defaults before project's build-extras.gradle
 if (!project.hasProperty('xwalkVersion')) {
-    ext.xwalkVersion = getConfigPreference("xwalkversion", DEFAULT_VERSION)
+    ext.xwalkVersion = getConfigPreference("CROSSWALK_ANDROID_VERSION", DEFAULT_VERSION)
```

Comment:
The README also need align with it.
xwalkVersion --------->  CROSSWALK_ANDROID_VERSION
xwalkCommandLine ---------> CROSSWALK_ANDROID_COMMANDLINE